app/src/main/java/com/rokt/roktdemo/ui/home/HomePage.kt

```diff
@@ -61,6 +64,13 @@ fun HomePage(actions: MainActions) {
             Footer()
             LargeSpace()
         }
+
+        SettingsCog(
```

Comment:
what is `Cog` means here ?

Reply:
Cog is the term used to refer `gear` icon.